arroyo/processing/strategies/__init__.py

```diff
@@ -3,5 +3,17 @@
     ProcessingStrategy,
     ProcessingStrategyFactory,
 )
+from arroyo.processing.strategies.collect import CollectStep, ParallelCollectStep
+from arroyo.processing.strategies.filter import FilterStep
+from arroyo.processing.strategies.transform import ParallelTransformStep, TransformStep
```

Comment:
I wonder if in the future we need to have a hierarchy under strategies or not. Ideally, these should be independently composable. So there shouldn't be any restrictions on how to compose them. Validations on composition could happen outside of this module

Reply:
We don't need one. Isn't that exactly what this PR does though? All of them are independently composable and exported in this one place here for easy usage.